### PR TITLE
enhancement(API): Updated Getcluster message

### PR DESCRIPTION
**Pull request by bishal7679**
Signed-off-by: bishal7679 <bishalhnj127@gmail.com>

Description:
Updated get-clusters command when its null

### CODE REVIEW

**Comment by bishal7679** [comment]
Its small changes!
@dipankardas011 PHAL!

**Comment by dipankardas011** [comment]
@AvineshTripathi or @kranurag7 you can review this PR

**Comment by dipankardas011** [comment]
Will not be able to contribute for 1 or 3 days here
sorry for the inconvience

**Comment by dipankardas011** [comment]
@bishal7679 whats your progress

**Comment by dipankardas011** [comment]
@bishal7679 are their any progress?

**Comment by bishal7679** [comment]
@dipankardas011 I got disconnected for my exam sorry for the long waiting response. In this pr only fmt message has been changed. Do u want any further changes??

**Review by karnatisrinivas — COMMENTED** [review]
Cosmetic changes, LGTM

File: cli/cmd/getCluster.go [hunk 1]
```diff
@@ -101,7 +101,12 @@ func Printer(i int) {
 	if err != nil {
 		panic(fmt.Errorf("JSON Convertion failed"))
 	}
-	printUtil(arr)
+	if len(toBePrinted) == 0 {
+		fmt.Println("No clusters found")
+	} else {
+		printUtil(arr)
+
+	}
```
Comment (dipankardas011) [inline cli/cmd/getCluster.go:109]: when the json.marshal converts a empty json it returns nil does that compares to len() == 0 ?

**Review by dipankardas011 — CHANGES_REQUESTED** [review]
Do check the comments

File: cli/cmd/getCluster.go (hunk 1 above)
Comment (dipankardas011) [reply cli/cmd/getCluster.go:109]: @bishal7679 you should check for nil as its of type []byte

Comment (dipankardas011) [reply cli/cmd/getCluster.go:109]: @bishal7679 have you tested this in local system?

**Review by dipankardas011 — CHANGES_REQUESTED** [review]
Require changes

**Comment by bishal7679** [comment]
@dipankardas011 Here it is working!
Mar-03-19-47

**Comment by dipankardas011** [comment]
got it

**Review by dipankardas011 — APPROVED** [review]
LGTM

**Comment by dipankardas011** [comment]
also @bishal7679 can you use the log.Err(""No cluster)

**Comment by dipankardas011** [comment]
> also @bishal7679 can you use the
> 
> ```go
> log.Err("No cluster")
> ```

use the logger package

**Comment by dipankardas011** [comment]
or else decide whether its a error or just a notice or Success message what are your opinion @AvineshTripathi

**Comment by bishal7679** [comment]
> or else decide whether its a error or just a notice or Success message what are your opinion @AvineshTripathi

it would be perfect I think!

**Comment by AvineshTripathi** [comment]
I think notice or successs should be better and not error

**Comment by dipankardas011** [comment]
@bishal7679 you can fix that I will merge the PR today 😄

**Pull request closed without merging**